app/assemblers/assemble_docs.rb

```diff
@@ -0,0 +1,19 @@
+class AssembleDocs
+  include Mandate
+
+  initialize_with :params
+
+  def call
+    SerializePaginatedCollection.(docs, serializer: SerializeDocs)
+  end
+
+  memoize
+  def docs
+    Document::SearchDocs.(
+      criteria: params[:criteria],
+      track_slug: params[:track_slug],
```

Comment:
For now, I've envisioned that people could either search on criteria (plaintext) or filter on a track. Whether we need the latter, 🤷